attachment_s3/README.rst

```diff
@@ -12,13 +12,30 @@ Activate S3 storage:
 * Create or set the system parameter with the key ``ir_attachment.location``
   and the value in the form ``s3``.
 
-Configure accesses with environment variables:
```

Comment:
The new table is nice, but there is no reference telling that this are environment variables to be configured. I would keep something similar to this row